content/en/docs/reference/labels-annotations-taints/_index.md

```diff
@@ -995,6 +995,19 @@ to record the version of Windows Server in use.
 
 The label's value is in the format "MajorVersion.MinorVersion.BuildNumber".
 
+### storage.alpha.kubernetes.io/migrated-plugins {#storagealphakubernetesiomigrated-plugins}
+
+Type: Annotation
+
+Example:`storage.alpha.kubernetes.io/migrated-plugins:" kubernetes.io/cinder"`
```

Comment:
```suggestion
Example:`storage.alpha.kubernetes.io/migrated-plugins: "kubernetes.io/cinder"
```

Reply:
Thanks for the reviews . i'll make the change